app/src/main/java/com/duckduckgo/app/browser/customtabs/DuckDuckGoCustomTabService.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.customtabs
+
+import android.net.Uri
+import android.os.Binder
+import android.os.Bundle
+import androidx.browser.customtabs.CustomTabsService
+import androidx.browser.customtabs.CustomTabsSessionToken
+import timber.log.Timber
+
+class DuckDuckGoCustomTabService : CustomTabsService() {
```

Comment:
To avoid isolated knowledge, would you mind adding some docs about how this works in our Docs? I’d also propose some Knowledge Sharing